src/components/tabs/index.tsx

```diff
@@ -4,13 +4,15 @@ import { Tab, TabButtonControls, TabDropdownControls } from './components'
 import { useOverflowRef, useTabItems, useSyncedTabGroups } from './hooks'
 import { TabItem, TabsProps } from './types'
 import s from './tabs.module.css'
+import TabNestingProvider, { useIsNested } from './helpers/tab-nesting-context'
 
 const Tabs = ({
   ariaLabel,
   ariaLabelledBy,
   children,
   initialActiveIndex = 0,
   showAnchorLine = true,
+  allowNestedStyles = false,
```

Comment:
🙋🏻 Is there ever a time we don't want to allow nested styles? Feels like something that would be easy to miss while using `Tabs` in code (not MDX).